packages/@aws-cdk-testing/cli-integ/lib/with-cdk-app.ts

```diff
@@ -331,6 +331,30 @@ export interface CdkModernBootstrapCommandOptions extends CommonCdkBootstrapComm
   readonly usePreviousParameters?: boolean;
 }
 
+export interface CdkGarbageCollectionCommandOptions {
+  /**
+   * The amount of days an asset should stay isolated before deletion, to
+   * guard against some pipeline rollback scenarios
+   *
+   * @default 0
```

Comment:
Can we make this default not 0? Maybe 30 days or something? 

IMO our defaults should ere on the side of rollbacks if possible

Reply:
perhaps 30 is too mch. just not 0 😅